tools/travis-test.sh

```diff
@@ -102,7 +105,7 @@ setup_chroot()
     python-dev python-nose python-pip cython"
 
   # faster operation with preloaded eatmydata
-  echo /usr/lib/libeatmydata/libeatmydata.so | \
+  echo '/usr/$LIB/libeatmydata.so' | \
```

Comment:
and it's odd that up there we had an `if` that decided what path to use for `libeatmydata.so`, but down here we always use `/usr/$LIB/libeatmydata.so`?